src/mbgl/renderer/paint_property_binder.hpp

```diff
@@ -222,9 +224,10 @@ class ConstantCrossFadedPaintPropertyBinder final
         return constantPatternPositions;
     }
 
-    std::size_t getVertexCount() const override { return 1; }
-    const /*std::tuple<ExpandToType<A, T>>&*/ void* getVertexValue(std::size_t) const override {
-        return &constantPatternPositions;
+    std::size_t getVertexCount() const override { return 0; }
+
+    std::tuple<ZoomInterpolatedVertexType<As>...> getVertexValue(std::size_t) const override {
+        return {ZoomInterpolatedVertexType<As>{0}...};
```

Comment:
@TimSylvester We don't need to send `constantPatternPositions` as vertex, you have it on the `uniformValue` method.  But I think we should decouple `evaluated` from `binders` so I would propose to compute the `constantPatternPositions` from evaluated.

Reply:
We keep `binders` for now only for vertex attributes and interpolation UBO. For other uniforms `evaluated` and other params, but without `binders` should be enough.